playbooks/freeipa.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: all
+  become: true
+  vars:
+    server_group: "server-{{ inventory_hostname }}"
+    server: "{{ groups[server_group][0] }}"
+  roles:
+    - etc_hosts
+    - katello_repositories
```

Comment:
This looks like it just sets up a freeipa server (and not a Capsule running a freeipa server) -- does it need the repositories if thats the case?